packages/firebase_remote_config/README.md

```diff
@@ -0,0 +1,75 @@
+# firebase_remote_config plugin
+
+A Flutter plugin to use the [Firebase Remote Config API](https://firebase.google.com/products/remote-config/).
+
+[![pub package](https://img.shields.io/pub/v/firebase_remote_config.svg)](https://pub.dartlang.org/packages/firebase_remote_config)
+
+For Flutter plugins for other Firebase products, see [FlutterFire.md](https://github.com/flutter/plugins/blob/master/FlutterFire.md).
+
+*Note*: This plugin is still under development, and some APIs might not be available yet. [Feedback](https://github.com/flutter/flutter/issues) and [Pull Requests](https://github.com/flutter/plugins/pulls) are most welcome!
+
+## Usage
+
+### Import the firebase_remote_config plugin
+To use the firebase_remote_config plugin, follow the [plugin installation instructions](https://pub.dartlang.org/packages/firebase_remote_config#pub-pkg-tab-installing).
+
+### Android integration
+
+Enable the Google services by configuring the Gradle scripts as such.
```

Comment:
This section seems relevant to every FlutterFire plugin and looks like every plugin does it a bit differently. firebase_messaging references the Firebase console assistant, while firebase_analytics links to the codelab. Yours seems to be copy-pasted from firebase_auth. Not a huge deal but perhaps we should open an issue to make this more consistent.

Reply:
Ack